app/views/students/edit.html.erb

```diff
@@ -0,0 +1,202 @@
+  <%= f.label :First_Name,class: "col-sm-4"%>
+    <div class="col-sm-8">
+      <%= f.text_field :first_name,class: "form-control"  , placeholder: "First_Name" %>
+         <div class='text-danger'>
+          <% if @student.errors.messages[:first_name].present? %>
+          <%=@student.errors.full_messages_for(:first_name).first%>
+          <% end %>
+        </div>
+    </div>
+</div>
+
+<div class="form-group <%= 'has-error' if @student.errors.messages[:middle_name].present? %>">
+  <%= f.label :Middle_Name,class: "col-sm-4"%>
+    <div class="col-sm-8">
+      <%= f.text_field :middle_name,class: "form-control"  , placeholder: "Middle_Name" %>
+        <div class='text-danger'>
+          <% if @student.errors.messages[:middle_name].present? %>
+          <%=@student.errors.full_messages_for(:middle_name).first%>
+          <% end %>
+        </div>
+    </div>
+</div>
+
+<div class="form-group <%= 'has-error' if @student.errors.messages[:last_name].present? %>">
+  <%= f.label :Last_Name,class: "col-sm-4"%>
+    <div class="col-sm-8">
+      <%= f.text_field :last_name,class: "form-control"  , placeholder: "Last_Name" %>
+         <div class='text-danger'>
+          <% if @student.errors.messages[:last_name].present? %>
+          <%=@student.errors.full_messages_for(:last_name).first%>
+          <% end %>
+        </div>
+    </div>
+  </div>
+
+  
+    <div class="form-group <%= 'has-error' if @student.errors.messages[:avatar].present? %>">
+  <%= f.label :avatar,class: "col-sm-4"%>
+    <div class="col-sm-8">
+      <%= f.file_field :avatar,class: "btn btn-default" %>
+         <div class='text-danger'>
+          <% if @student.errors.messages[:avatar].present? %>
+          <%=@student.errors.full_messages_for(:avatar).first%>
+          <% end %>
+        </div>
+    </div>
+  </div>
+  
+  <div class="form-group <%= 'has-error' if @student.errors.messages[:gender].present? %>">
+  <%= f.label :Gender,class: "col-sm-4"%>
+    <div class="col-sm-8">
+<div class="radio radio-inline">
+    <%=f.radio_button(:gender, "Male") %>
+    <%= f.label(:gender_male, "Male") %></div>
+    <div class="radio radio-inline">
+    <%= f.radio_button(:gender, "Female") %>
+    <%= f.label(:age_adult, "Female") %></div>
+         <div class='text-danger'>
+          <% if @student.errors.messages[:gender].present? %>
+          <%=@student.errors.full_messages_for(:gender).first%>
+          <% end %>
+        </div>
+    </div>
+  </div>
+  
+</div>
+</div>
+</div>
+</div>
+</fieldset>
+
+<div class="col-md-6">
+<div class="login-panel panel panel-default">
+<div class="panel-body">
+ <div class="panel-form">
+<fieldset>
+
+<div class="form-group <%= 'has-error' if @student.errors.messages[:date_of_birth].present? %>">
+  <%= f.label :Date_Of_Birth,class: "col-sm-4"%>
+    <div class="col-sm-8">
+      <%= f.text_field :date_of_birth,class: "form-control datepicker",placeholder: "DD-MM-YYYY" %>
```

Comment:
add one space after ,(comma).
remove space before ,(comma).

you have to follow ruby style guide.
check all files and change accordingly.